src/main/java/com/aws/greengrass/certificatemanager/certificate/CertificateRequestGenerator.java

```diff
@@ -52,21 +46,28 @@ private CertificateRequestGenerator() {
      * @throws IOException fails to add SAN Extension to CSR builder
      */
     public static String createCSR(KeyPair keyPair,
-                                                  String thingName,
-                                                  List<InetAddress> ipAddresses,
-                                                  List<String> dnsNames) throws OperatorCreationException, IOException {
+                                   String thingName,
+                                   List<InetAddress> ipAddresses,
+                                   List<String> dnsNames) throws OperatorCreationException, IOException {
+        // TODO: Remove once Moquette and Bridge no longer call this API
+        return createCSR(keyPair, thingName);
+    }
 
+    /**
+     * Returns a PEM encoded Certificate Signing request string for given connectivity info.
+     *
+     * @param keyPair     public/private key pair to generate cert
+     * @param thingName   common name for cert subject
+     * @return String
+     * @throws OperatorCreationException fails to build CSR content signer with the given private key
+     * @throws IOException fails to add SAN Extension to CSR builder
+     */
+    public static String createCSR(KeyPair keyPair, String thingName) throws OperatorCreationException, IOException {
         // Create PKCS10 certificate request
         X500Principal x500Principal = getx500PrincipalForThing(thingName);
         PKCS10CertificationRequestBuilder p10Builder = new JcaPKCS10CertificationRequestBuilder(
                 x500Principal, keyPair.getPublic());
 
-        // Add SAN cert extensions to request
-        ExtensionsGenerator extensions = getSANExtensions(ipAddresses, dnsNames);
```

Comment:
we don't need any SAN?

Reply:
The certificate generator overwrites this information anyway. This is leftover from early days before we had the connectivity information stuff ready.

https://github.com/aws-greengrass/aws-greengrass-client-device-auth/blob/fd5003a0aa74af7f4abead417595d58302ce5f24/src/main/java/com/aws/greengrass/certificatemanager/certificate/CertificateHelper.java#L244